test/unit/components/reveal.test.js

```diff
@@ -0,0 +1,45 @@
+const expectComponent = require('../component-helper').expectComponent;
+
+describe('Reveal component', () => {
+  it('should render', () => {
+    expectComponent(
+      'reveal',
+      {
+        variant: 'block',
+        summary: 'Summary text',
+        children: [],
+      },
+      `<details class="">
+        <summary data-analytics="summary">
+          <span class="details__summary">
+      Summary text
+    </span>
```

Comment:
I tried [js-beautify](https://github.com/beautify-web/js-beautify) and it seems to work a bit better because it has some additional useful options such as `max_preserve_lines` which allows to remove any unnecessary whitespace.

I've [converted the tests for you](https://github.com/nhsuk/betahealth/compare/component-loading...component-loading-test-formatting). I think it's an improvement because HTML formatting is more sensible. What do you think?